### ISSUE TITLE
Is there a 3D softmax function?

### ISSUE THREAD

**Issue by dasha50** [pytorch/pytorch#3915]
My pytorch version is 0.2.0_3, and i need to process a 3D tensor by softmax. But  torch.nn.softmax() need a 2D tensor. Who konws how to solve the problem?

**Comment by soumith** [comment]
https://discuss.pytorch.org
